content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Standalone Operator Quick Start Evaluation Install [Experimental]
+description: Instructions to install Istio in a Kubernetes cluster for evaluation.
+weight: 11
+keywords: [kubernetes, operator]
+aliases:
+---
+
+This guide installs Istio using the standalone Istio operator. The only dependencies
+required are a supported Kubernetes cluster and the `kubectl` command. This
+installation method lets you quickly evaluate Istio in a Kubernetes cluster on
+any platform using a variety of profiles.
+
+To install Istio for production use, we recommend using the [Helm Installation guide](/docs/setup/install/helm/)
+instead, which is a stable feature.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/#downloading-the-release).
```

Comment:
no longer needed, right? or should we still encourage it to get istioctl

Reply:
You may be correct removing may be the best option.  We do want people to download istioctl, but everytime someone wants me to install a downloaded binary in my path, I do this:

https://images.genius.com/2e17391a56a8c0441686b577da3678c9.500x376x35.gif

IOW the overhead of installing a binary slows people down from evaluating Istio.  Add to that tar.  I know these seem like trivial things, but the way these projects typically work out is a CTO or CIO office is asked to evaluate a project.  If they have do these "standard magic incantations" to get a working system, they have a negative impact on the resulting white paper.  I know for certain - I've written several while working in a CTO office 👍